scinet/hyperparameters.py

```diff
@@ -23,3 +23,5 @@ def __init__(self):
 
         self.optimizer = optim.Adam
         self.leadingLoss = nn.MSELoss
+
+        self.annealEpoch = 0
```

Comment:
I'm going to make this default to None, instead of 0, just to be more clear.
It took me a while staring at this and the fact that in all your examples you immediately crank it way up to realize that this is just a flag to not use beta-annealing at all, and the value 0 has no special mathematical meaning at all.
Using None will just make this more explicit.